--- OWRootHandler.cs	2016-06-03 20:35:38.000000000 +0300
+++ OWRootHandler.cs	2016-06-21 11:12:44.000000000 +0900
@@ -93,7 +93,7 @@ namespace CASCExplorer

                 string name = filedata[4];

-                if (Path.GetExtension(name) == ".apm" && name.Contains("LenUS"))
+                if (Path.GetExtension(name) == ".apm" && name.Contains("LenUS") && name.Contains("RCN"))
                 {
                     // add apm file for dev purposes
                     ulong apmNameHash = Hasher.ComputeHash(name);
